drivers/arm/ledbbtwi.c

```diff
@@ -0,0 +1,9 @@
+#include "ledbbtwi.h"
```

Comment:
Better to remove (and the header) and let the build fail, as its not like we have the i2c ps2avrgb implementation copied to ARM.

Reply:
Can do

Reply:
Actually, will rip out entire ledbbtwi separation for now, in favor of RGBW support